src/components/Page/styles/Card.css.ts

```diff
@@ -45,19 +44,19 @@ export const CardUI = styled('div')`
   }
 
   ${breakpoint(
-    PageConfig.breakpoint.widescreen,
```

Comment:
Looks like some formatting went wonky here. No big deal at all! (For real, haha).

If prettier can't catch/autofix it, then it doesn't matter (IMO). Not worth debating these things 🤗

Reply:
Ahhh, not sure why my editor didn't catch some of these, just ran prettier and it did find some other places and fixed them, I remember adding a few easy steps so prettier runs always on precommit, which was very handy in case people didn't have their editor to do it automatically on save :)